.github/workflows/deploy_release_to_marketplace.yml

```diff
@@ -0,0 +1,48 @@
+name: autokittehVSCode CI - Publish to VSCode Marketplace
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
```

Comment:
It's not environment, maybe release type

Reply:
renamed